method/maple/maple.r

```diff
@@ -174,7 +174,7 @@ seurat_obj@images$image =  new(
   Class = 'SlideSeq',
   assay = "Spatial",
   key = "coordinates_",
-  coordinates = as.data.frame(spatialCoords(spe))
+  coordinates = as.data.frame(colData(spe))
```

Comment:
Cordinates are stored in the spatialCoords, in the colData you will only find array rows/cols for ST/Visium datasets. Is this intended?

Either way you should probably explicitly extract the relvant columns from colData as there could be more in the future